### PR TITLE
Got first Spock test working: Hello World example

### PR DESCRIPTION
**Pull request by djangofan**
Hi, a long time back I said I was interested in unit testing your pluging  (see https://github.com/jmeter-gradle-plugin/jmeter-gradle-plugin/issues/16  )   and so, I am creating this pull request for you to look at to see if I am going in the right direction for you?

### CODE REVIEW

**Comment by foragerr** [comment]
@djangofan Thanks for helping out with tests! I'm not an expert in the area and cant give you a lot of direction. Just a few quick high level thoughts, feel free to tell me I'm wrong:

* Spock definitely is the right way to go.
* gradle test kit is what other gradle plugins are using to build tests
* Maybe you should not add idea specific config to the repo. Feel free to use it locally but don't check them in

**Comment by romankh3** [comment]
I'm using Spoke in on of my projects and I want to say that this framework is deeper than JUnit4/5 for testing. It's a BDD concept.

File: src/test/groovy/net/foragerr/jmeter/gradle/plugins/HelloSpockSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import spock.lang.*
+
+class HelloSpockSpec extends spock.lang.Specification {
```
Comment (romankh3) [inline src/test/groovy/net/foragerr/jmeter/gradle/plugins/HelloSpockSpec.groovy:3]: I think import will be better.

File: build.gradle [hunk 2]
```diff
@@ -20,13 +22,28 @@ version = "$thisPluginVersion-$jmVersion-SNAPSHOT"
...
 //--- Dependencies ---
 dependencies {
 	compile gradleApi()
-    compile	localGroovy()
+	//compile localGroovy()
+    groovy 'org.codehaus.groovy:groovy-all:2.4.7'
+	
+	testCompile(
+            'junit:junit:4.12',
+            'org.spockframework:spock-core:1.1-groovy-2.4-rc-3',
```
Comment (romankh3) [inline build.gradle:43]: you can update version of the Spoke to `1.1-groovy-2.4`. Now it's available.

**Review by romankh3 — CHANGES_REQUESTED** [review]
There is a good idea to add Spoke for testing!
For it needs work. Also, the build is red.

**Pull request closed without merging**